Libraries/LibWebView/BrowserProcess.h

```diff
@@ -19,6 +20,8 @@
 
 namespace WebView {
 
+using namespace Messages::UIProcessServer;
```

Comment:
Please no

Reply:
To expand on this, `using namespace` in header files is strictly verboten. And in cpp files it's discouraged, with a few specific exceptions (AK::TimeLiterals for example).